src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
```suggestion
        Machine truck = new Truck();
```
краще для типу посилання використовувати абстрактний тип (інтерфейс або абстрактний клас) це дозволить ефективніше використовувати поліморфізм і заміняти одну реалізацєю іншою, наприклад
```java
Machine workingMachine = truck;
workingMachine.doWork();
workingMachine = excavator;
workingMachine.doWork();
....
```